<ui version="4.0" >
 <author>Volker Krause &lt;vkrause@kde.org></author>
 <class>ItemViewWidget</class>
 <widget class="QWidget" name="ItemViewWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Model:</string>
     </property>
     <property name="buddy" >
      <cstring>modelBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="modelBox" />
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QTreeView" name="itemView" >
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <includes/>
 <resources/>
 <connections/>
</ui>
